Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+### 1. Describe some of the differences between `.forEach` & `.map`.
+#### Some of the differences between `.forEach` & `.map` are as follows:
+#### `.forEach` scours an array for a particular item in the array. Once it finds the index it allows us to make use of that item but it doesn't return that item the way `.map` would.
+#### `.map` iterates over an array and breaks it up into individual parts and then returns a brand new array at the end of the sequence.
+
+### 2. Name five different Types in JavaScript. A Type is something that can represent data. What is so special about Arrays?
+#### Undefined, null, boolean, number, string are all types that represent different data points in JS.
+#### An array is a special data type called an object. It is considered an object because it acts like an object in the sense that it is a container for different values.
```

Comment:
In JavaScript this is true, but there may be a fundamental misunderstanding between the difference between objects and arrays. Maybe elaborate further to me on slack about this one.